app/workers/downstream_live_worker.rb

```diff
@@ -14,8 +14,10 @@ class DownstreamLiveWorker
   def self.uniq_args(args)
     [
       args.first.fetch("content_item_id"),
+      args.first.fetch("message_queue_update_type", nil),
+      args.first.fetch("update_dependencies", true),
       'live'
-    ]
+    ].compact
```

Comment:
niggle but I think this would be better without the compact here since they'd always have arguments rather than a varying number